tests/jest-puppeteer.config.js

```diff
@@ -4,8 +4,9 @@
  */
 module.exports = {
   launch: {
-    headless: true,
-    devtools: false,
+    headless: !!process.env.CI,
+    devtools: !process.env.CI,
+    slowMo: 100, // slow down by 100ms to mimick real world usage (even if that slows down test suite...)
```

Comment:
Up until now we were testing with 0 lag, which was creating strange artifacts because our UI has a little latency (such as when the user clicks on a button... etc). This should _not_ change the behavior but is more aligned with actual user behavior!